django/contrib/gis/db/models/fields.py

```diff
@@ -272,7 +272,9 @@ def select_format(self, compiler, sql, params):
         of the spatial backend. For example, Oracle and MySQL require custom
         selection formats in order to retrieve geometries in OGC WKB.
         """
-        return compiler.connection.ops.select % sql, params
+        if not compiler.query.subquery:
```

Comment:
Should we do the same in `ExtentField.select_format()`? :thinking:

Reply:
@felixxm from my local testing doing so causes some `gis` tests to fail on all non-Postgres backends.